.travis.yml

```diff
@@ -17,7 +17,7 @@ before_script:
     - sh -c "if [ '$SYMFONY_DEPS_VERSION' = '' ]; then sed -i 's/~2\.8|^3\.0/2.8.*@dev/g' composer.json; composer update; fi"
     - composer install
 
-script: phpunit
+script: SYMFONY_DEPRECATIONS_HELPER=weak phpunit
```

Comment:
btw, please use the ``env: global: ...`` config to set the variable instead. This way, it can be overwritten in the matrix